.github/workflows/build-release.yml

```diff
@@ -13,18 +13,17 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - { name: "3.8", python: "3.8", tox: py38 }
-          - { name: "3.12", python: "3.12", tox: py312 }
-          - { name: "lowest", python: "3.8", tox: py38-lowest }
```

Comment:
i think we want to keep the `lowest` build, to test against the lowest supported python+nltk version. so this will need to be changed to `tox: py39-lowest`

Reply:
@sloria Does `lowest` look okay to you now?

Reply:
yes 👍